### DISCUSSION TITLE
Getting a new error in filament panel

### DISCUSSION THREAD

**Discussion by codingwithrk** [filamentphp/filament#11455] [Help]
### Package

Notifications

### Package Version

v3.2

### How can we help you?

This is my code: https://github.com/CodingwithRK/sharecode

This is an error:
**_Navigation group [Blog] has an icon but one or more of its items also have icons. Either the group or its items can have icons, but not both. This is to ensure a proper user experience._**

Hi friends, when I mentioned the icon in the navigation group it was throwing an error.

**Comment by leandrocfe** [comment]
You can't use both. What about [clusters](https://filamentphp.com/docs/3.x/panels/clusters)?
